src/Symfony/Component/Security/Core/User/MissingUserProvider.php

```diff
@@ -18,6 +18,8 @@
  * when a firewall requires a user provider but none was defined.
  *
  * @internal
+ *
+ * @template-implements UserProviderInterface<UserInterface>
```

Comment:
Does Psalm allow us to do this?

```suggestion
 * @template-implements UserProviderInterface<never>
```

Reply:
Seems like it does ; I made the change